### ISSUE TITLE
Some smaller issues

### ISSUE THREAD

**Issue by fabiand** [kubevirt/kubevirt-ansible#183]
These two issues made it hard to follow the readme:

- WRT selinux it doesn't specify what "inventory" to run (e.g. let's specify it's the openshift-ansible inventory)
- The "KubeVirt parameters documentation" link is broken.

**Comment by alexxa** [comment]
@fabiand Do you mean that `ansible-playbook -i inventory playbooks/selinux.yml` will not work with our [inventory](https://github.com/kubevirt/kubevirt-ansible/blob/master/inventory) and it should be one of [openshift-ansible inventories](https://github.com/openshift/openshift-ansible/tree/master/inventory)?

**Comment by alexxa** [comment]
@yuvalif Please let me know if you are ok with PR #187. Thanks.

**Comment by yuvalif** [comment]
commented in the review

**Comment by alexxa** [comment]
Fixed in PR #187.
